Person_rb

```diff
@@ -1,41 +1,27 @@
-# frozen_string_literal: true
```

Comment:
- [ ] This file name should be `person.rb`, not `Person_rb`. It should have the `.rb` file extension